Sources/Commands/init.swift

```diff
@@ -182,11 +182,11 @@ final class InitPackage {
 enum InitMode: CustomStringConvertible {
     case library, executable
 
-    init(_ rawValue: String?) throws {
-        switch rawValue?.lowercased() {
-        case "library"?, "lib"?:
+    init(_ rawValue: String) throws {
+        switch rawValue.lowercased() {
+        case "library", "lib":
             self = .library
-        case nil, "executable"?, "exec"?, "exe"?:
+        case "executable", "exec", "exe":
```

Comment:
I agree, though I have to admit that `exec` and `lib` are very tempting shortcuts.  But this might not be super-important — `init` might not be used frequently enough to warrant the abbreviations.